### DISCUSSION TITLE
What adding a better internet test?

### DISCUSSION THREAD

**Discussion by Manu99it** [ChristopherKlay/StadiaEnhanced#113] [Ideas & Requests]
Actually the official stadia speed test isn't good enough to effectively tell if you can have at least an acceptable experience. Well, it's just a speed test. What adding a link or a hint to a packet loss test like [this](https://packetlosstest.com/) ? It also explain things well enough to be understandable by majority of people

**Comment by ChristopherKlay** [comment]
> Actually the official stadia speed test isn't good enough to effectively tell if you can have at least an acceptable experience.

Just like the one you linked can't effectively tell you if your connection to Googles servers (and thus your experience with the service) will be good.

The test isn't supposed to tell you how well Stadia runs on your end - because it simply can't predict that in the first place. Those tests don't check your connection to the server that actually matters, they can't check how well the incoming stream is decoded (which is by far the biggest source of issues) and so on - bandwidth is only a very small part of what actually matters.

The last thing we need is a more detailed connection test that shows more information about a connection that while being affected by general issues (i.e. weak wifi) doesn't matter at all for the actual stream. Best case people get inaccurate data compared to just registering for free and checking their connection via the free games (Crayta, Destiny and Bomberman) and the streaming monitor (which shows the data of the actual stream, including packet loss and such) - worst case it would just make people believe Stadia for example provides a bad service "because this test says my internet is fine" when the actual experience (due to several possible issues) isn't.

I'd gladly replace it with a better working one specifically designed for Stadia, but currently the only official test related to (and used by) Google is the M-Lab based one, that's currently linked (which is the same, compared to actually googling "internet speed test").
